pytorch/inference/docker/1.3.1/py3/Dockerfile.eia

```diff
@@ -18,7 +18,10 @@ ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
 ENV TEMP=/home/model-server/tmp
 
 RUN apt-get update \
- && apt-get install -y --allow-downgrades --allow-change-held-packages --no-install-recommends \
+    && apt-get install -y --no-install-recommends software-properties-common \
+    && add-apt-repository ppa:openjdk-r/ppa \
+    && apt-get update \
+    && apt-get install -y --allow-downgrades --allow-change-held-packages --no-install-recommends \
```

Comment:
Could you please explain this line, and also add a comment as it can potentially affect other packages?

Reply:
The intention is to install JDK 11 as its need by TS. 

1. Install software-properties-common for add-apt-repository
2. JDK 11 is not available for Ubuntu 16.04 in the repos. Hence had to use PPA.